cli/tests/integration_tests.rs

```diff
@@ -326,10 +326,86 @@ fn bundle_single_module() {
   assert_eq!(output.stderr, b"");
 }
 
-// TODO(#2933): Rewrite this test in rust.
 #[test]
 fn repl_test() {
-  util::run_python_script("tools/repl_test.py")
+  use std::io::Write;
+  use std::process::Stdio;
+  let repl_process = |lines: &Vec<&str>,
+                      envs: Option<Vec<(String, String)>>,
+                      stdout_str: &str,
+                      stderr_str: &str,
+                      exit_code: i32| {
+    let mut deno_process_builder = util::deno_cmd();
+    deno_process_builder
+      .stdin(Stdio::piped())
+      .stdout(Stdio::piped())
+      .stderr(Stdio::piped());
+    if let Some(envs) = envs {
+      deno_process_builder.envs(envs);
+    }
+    let mut deno = deno_process_builder
+      .spawn()
+      .expect("failed to spawn script");
+    {
+      let stdin = deno.stdin.as_mut().expect("failed to get stdin");
+      stdin
+        .write_all(lines.join("\n").as_bytes())
+        .expect("failed to write to stdin");
+    }
+    let output = deno.wait_with_output().expect("failed to wait on child");
+    assert!(util::pattern_match(
+      stdout_str,
+      std::str::from_utf8(&output.stdout).unwrap(),
+      "[WILDCARD]"
+    ));
+    assert!(util::pattern_match(
+      stderr_str,
+      std::str::from_utf8(&output.stderr).unwrap(),
+      "[WILDCARD]"
+    ));
```

Comment:
These two pattern_match don't seem to do anything? `[WILDCARD]` matches everything.